src/index.ts

```diff
@@ -71,21 +73,23 @@ async function processSqsRecord(sqsRecord: SQSRecord) {
         const sourceObjectKey = decodeURIComponent(s3Record.s3.object.key.replace(/\+/g, ' '))
 
         const params = {
-          "Bucket": sourceBucketName,
-          "Key": sourceObjectKey
+          Bucket: sourceBucketName,
+          Key: sourceObjectKey,
         }
         debug(`Getting S3 object ${JSON.stringify(params)}`)
 
         const command: GetObjectCommand = new GetObjectCommand(params)
-        const {Body} = await s3Client.send(command)
+        const { Body } = await s3Client.send(command)
 
         debug(`Sending logs to Firehose`)
         await sendLogsToFirehose(sourceBucketName, sourceObjectKey, Body as Readable)
-      } else {
+      }
+      else {
```

Comment:
hmm, we don't do `else` on a new line. I think this will be undone by IDE autoformatting

Reply:
We need to write some custom rules for eslint then, this was the eslint auto fix